docs/mpcie_modules.md

```diff
@@ -207,6 +207,110 @@ can be done, for example in Debian OS, by performing those steps:
     sudo apt-get install firmware-realtek
     ```
 
+## SATA controllers
+
+
+### ASMedia ASM1061
+
+ASM1061 was tested with Delock 95233 mPCIe 2xSATA module.
+
+![ASM1061](images/delock_95233.png)
+
+**Compatible slots**
+
+Due to apu boards construction, it is possible to test that module only under
+apu2 mPCIe2 slot. Slot mPCIe1 at any platform is unreachable. SATA connectors in
+Delock 95233 are on the side, which makes it impossible to attach hard drive
+(capacitors next to mPCIe1 slot block it).
+
```

Comment:
I would type here that the module is not supported for now.

The test configurations and possible reasons section should land to the docs/debug directory of this repository.

When You create such file, please link it in this document, so anyone who reads this will check why the module is not supported.

Reply:
Move it to docs/debug/mpcie2.md and created 2 sections there: Tests results and Possible reasons.